### PR TITLE
created post method for posts

### PR DESCRIPTION
**Pull request by natapoli90**

### CODE REVIEW

**Comment by billhimmelsbach** [comment]
Great work team!

**Comment by nathanallen** [comment]
I heard you like posts, so i made a posts endpoint for you to post posts to.

File: vagabond/app/controllers/city_posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class CityPostsController < ApplicationController
+
+  def new
+    @post = Post.new
+    render :new
+  end
+
+  def create
```
Comment (nathanallen) [inline vagabond/app/controllers/city_posts_controller.rb:8]: Gonna want some auth on this action!

File: vagabond/app/controllers/city_posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    @city = City.find(params[:city_id])
+    post = Post.create(post_params)
+    @city.posts.push(post)
```
Comment (nathanallen) [inline vagabond/app/controllers/city_posts_controller.rb:11]: `@city.posts.create(post_params)` works too!

**Pull request merged**